src/shipit_risk_assessment/requirements_frozen.txt

```diff
@@ -29,3 +29,4 @@ taskcluster==1.3.3
 urllib3==1.21.1
 whatthepatch==0.0.5
 yarl==0.10.2
+flake8-coding
```

Comment:
this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``